docs/modules/clusters/pages/ucn-dynamic-config.adoc

```diff
@@ -0,0 +1,47 @@
+= Dynamic Configuration API
+:description: You can use the dynamic configuration API to deploy resources from your client. This means that you can configure namespaces and their resources, and replace the resources at runtime for use when a user customization is next instantiated.
+:page-enterprise: true
+:page-beta: true
+
+{description}
+
+For example, you can use the dynamic configuration API to add or remove a specific namespace from a running cluster's configuration.
```

Comment:
> remove a specific namespace

We don't support removal of a namespace, we support add or replace. So maybe replace would be a better word than remove?